libraries/dawg-show/src/main/java/com/comcast/video/dawg/show/key/KeyController.java

```diff
@@ -44,17 +51,26 @@
     @Autowired
     private RemotePluginManager remotePluginManager;
 
+    @Autowired
+    private JwtDeviceAccessValidator accessValidator;
+    
+    private DawgCookieUtils cookieUtils = new DawgCookieUtils();
+
     @RequestMapping(method = { RequestMethod.POST }, value = "/sendKey")
     @ResponseBody
-    public void sendKey(@RequestParam(value="deviceIds[]") String[] deviceIds, @RequestParam String key, @RequestParam(required = false) String remoteType) throws KeyException, InterruptedException {
-        sendKey(deviceIds, key, null, remoteType);
+    public void sendKey(HttpServletRequest req, HttpServletResponse resp,
+            @RequestParam(value="deviceIds[]") String[] deviceIds, @RequestParam String key, @RequestParam(required = false) String remoteType,
+            @CookieValue(name="dawt", required=false) String dawt) throws KeyException, InterruptedException {
```

Comment:
i think `@CookieValue` should be referencing `DawgCookieUtils.COOKIE_NAME` instead